spinn_front_end_common/utility_models/data_speed_up_packet_gatherer_machine_vertex.py

```diff
@@ -430,16 +457,29 @@ def get_data(
             self._write_routers_used_into_report(
                 self._report_path, routers_been_in_use, placement)
 
+        # Stop anything else getting through
```

Comment:
dont you want this at line 447??? to ensure minimal time betwene finish recieve and extra sends?